testsuite/MDAnalysisTests/coordinates/test_lammps.py

```diff
@@ -115,6 +115,32 @@ def LAMMPSDATAWriter(request, tmpdir_factory):
     return u, u_new
 
 
+@pytest.fixture(params=[
+    [LAMMPSdata, True],
+    [LAMMPSdata_mini, True],
+    [LAMMPScnt, True],
+    [LAMMPShyd, True],
+    [LAMMPSdata, False]
+], scope='module')
+def LAMMPSDATAWriter_molecule_tag(request, tmpdir_factory):
+    filename, charges = request.param
+    u = mda.Universe(filename)
+    if charges == False:
```

Comment:
```suggestion
    if not charges:
```

Reply:
Please review https://github.com/MDAnalysis/mdanalysis/actions/runs/4859346505/jobs/8661858282 this was highlighted by the darker linting bot as a flake8 failure (see: https://github.com/MDAnalysis/mdanalysis/pull/4114#issuecomment-1496674113)

Reply:
Oh, thanks again to @RMeli. Flake8 should be pass fine now.